File: Java/Car.java
package Java;

public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passagenger;

    void pintDataCar(){
        System.out.println("Car" + this.license + " Driver " + driver.name);
    }

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }
}

File: JS/Car.js
class Car {

    constructor(licence, driver) {
        this.id;
        this.license = licence;
        this.driver = driver;
        this.passenger;
    }
    
    printCar = function () {
        console.log(this.driver);
        console.log(this.driver.name);
        console.log(this.driver.document);
    }
}